### ISSUE TITLE
[wily] Some gnome 3.18 ebuilds break vivid under ~amd64 

### ISSUE THREAD

**Issue by and3k** [shiznix/unity-gentoo#127]
Same as #113, but for wily.

These masks work for me:

```
=gnome-*/*-*18*::gentoo
=app-editors/gedit-3.18*::gentoo
=app-text/evince-3.18*::gentoo
=mail-client/evolution-3.18*::gentoo
=media-video/cheese-3.18*::gentoo
=media-video/totem-3.18*::gentoo
=www-client/epiphany-3.18*::gentoo
=x11-libs/gtksourceview-3.18*::gentoo
=app-text/yelp-tools-3.18*::gentoo
=media-plugins/grilo-plugins-0.2.16::gentoo
=media-plugins/grilo-plugins-0.2.17::gentoo
```

Thanks,
Bela

**Comment by shiznix** [comment]
If you are experiencing gnome-3.18 packages being pulled in for Wily, can please you pastebin the 'emerge -pvt ...' deptree output to show what packages are pulling 3.18.

For stable Unity (currently Wily) we rely on Gnome's ~arch keyword mask to pull in stable Gnome (currently 3.16).
This is one of the reasons why the overlay also does not support a globally set unstable ~arch.

**Comment by and3k** [comment]
Yup, I know ~amd64 is not supported. But if I remember correctly, the new gnome version normally gets set to stable in Gentoo before the next unity version comes out, so I thought it would be helpful to mask it now already. But I’m fine if you want so close this bug :) I have my working mask file anyway.

I will post the emerge output later, I can’t right now.

**Comment by shiznix** [comment]
Thanks whenever you can :)

You're right it should be masked now in preparation for 3.18 going stable instead of waiting for it to break.
Will do that shortly...

**Comment by and3k** [comment]
Now that I have updated my system with my mask file, I seems emerge -pvt doesn’t give me the full tree… here is the output I get when I **don’t mask** gnome 3.18:

`emerge -ptvuDN --with-bdeps=y --verbose-conflicts unity-meta`:

```
These are the packages that would be merged, in reverse order:

Calculating dependencies... done!

!!! All ebuilds that could satisfy ">=net-libs/gnome-online-accounts-3.17.91" have been masked.
!!! One of the following masked packages is required to complete your request:
- net-libs/gnome-online-accounts-3.18.3::gentoo (masked by: package.mask)
- net-libs/gnome-online-accounts-3.18.2.1::gentoo (masked by: package.mask)

(dependency required by "media-plugins/grilo-plugins-0.2.17::gentoo[gnome-online-accounts]" [ebuild])
(dependency required by "media-video/totem-3.18.1::gentoo" [ebuild])
(dependency required by "gnome-base/gnome-core-apps-3.16.0::unity-gentoo" [installed])
(dependency required by "unity-base/unity-meta-15.10::unity-gentoo[gnome]" [installed])
(dependency required by "unity-meta" [argument])
For more information, see the MASKED PACKAGES section in the emerge
man page or refer to the Gentoo Handbook.
```

**Comment by shiznix** [comment]
Should be fixed in commit eeade944a1075577755f84c41abc09665c53d511

Thanks :)
